addons/api/addon/models/worker.js

```diff
@@ -1,27 +1,48 @@
 import GeneratedWorkerModel from '../generated/models/worker';
-import { fragment } from 'ember-data-model-fragments/attributes';
+import { attr } from '@ember-data/model';
 
 export default class WorkerModel extends GeneratedWorkerModel {
   // =attributes
 
-  /**
-   * @type {[FragmentTagModel]}
-   */
-  @fragment('fragment-tag', {
+  @attr({
     description:
       'The deduplicated union of the tags reported by the worker ' +
       'from its configuration and any tags added through other means.\nOutput only.',
     readOnly: true,
   })
   canonical_tags;
 
-  /**
-   * @type {[FragmentTagModel]}
-   */
-  @fragment('fragment-tag', {
+  @attr({
     description:
       "The tags set in the worker's configuration file.\nOutput only.",
     readOnly: true,
   })
   config_tags;
+
+  save(options) {
+    if (this.isNew && this.isPki) {
```

Comment:
Where is `isPki` defined?

Reply:
We are not including this conditional check anymore